src/tools/CMakeLists.txt

```
@@ -69,6 +69,39 @@ if(AXOM_ENABLE_QUEST)
         blt_add_target_compile_flags(TO mesh_tester FLAGS "-fgpu-rdc")
         blt_add_target_link_flags(TO mesh_tester FLAGS "-fgpu-rdc")
     endif()
+
+    # Add unit tests
+    if(AXOM_ENABLE_TESTS AND AXOM_DATA_DIR AND RAJA_FOUND AND UMPIRE_FOUND)
+
+        # Run the mesh_tester with the plane_simp_problems.stl example and
+        # different spatial indexes, raja policies
+
+        set(plane_data_dir ${AXOM_DATA_DIR}/quest/plane_simp_problems.stl)
```

```suggestion
        set(plane_data_dir "${AXOM_DATA_DIR}/quest/plane_simp_problems.stl")
```
Should be quoted for windows if there is a space

---

Thanks for catching that